### PR TITLE
feat: Support selector on describe

### PR DESCRIPTION
**Pull request by bbernays**
```
cloudquery % go run main.go policy describe aws//foundational_security
Describe Policy aws output:

+----------------------------------------------+--------------------------------+
|                     PATH                     |          DESCRIPTION           |
+----------------------------------------------+--------------------------------+
| aws//foundational_security                   | AWS Foundational Security Best |
|                                              | Practices controls             |
| aws//foundational_security/acm               | acm controls                   |
| aws//foundational_security/apigateway        | apigateway controls            |
| aws//foundational_security/autoscaling       | autoscaling controls           |
| aws//foundational_security/cloudfront        | cloudfront controls            |
| aws//foundational_security/cloudtrail        | cloudtrail controls            |
| aws//foundational_security/codebuild         | codebuild controls             |
| aws//foundational_security/config            | AWS config controls            |
| aws//foundational_security/dms               | AWS DMS controls               |
| aws//foundational_security/dynamodb          | DynamoDB controls              |
| aws//foundational_security/ec2               | EC2 controls                   |
| aws//foundational_security/ecs               | ECS controls                   |
| aws//foundational_security/efs               | EFS controls                   |
| aws//foundational_security/elastic_beanstalk | ElasticBeanstalk controls      |
| aws//foundational_security/elb               | ELB controls                   |
| aws//foundational_security/elbv2             | ELBv2 controls                 |
| aws//foundational_security/emr               | EMR controls                   |
| aws//foundational_security/elasticsearch     | Elasticsearch controls         |
| aws//foundational_security/guardduty         | GuardDuty controls             |
| aws//foundational_security/iam               | IAM controls                   |
| aws//foundational_security/kms               | KMS controls                   |
| aws//foundational_security/lambda            | Lambda controls                |
| aws//foundational_security/rds               | RDS controls                   |
| aws//foundational_security/redshift          | Redshift controls              |
| aws//foundational_security/s3                | S3 controls                    |
| aws//foundational_security/sagemaker         | SageMaker                      |
| aws//foundational_security/secretsmanager    | SecretsManager controls        |
| aws//foundational_security/sns               | SNS                            |
| aws//foundational_security/sqs               | SQS                            |
| aws//foundational_security/ssm               | SSM controls                   |
| aws//foundational_security/waf               | WAF controls                   |
+----------------------------------------------+--------------------------------+
To execute any policy use the path defined in the table above.
For example `cloudquery policy run aws//cis_v1.2.0`
```

### CODE REVIEW

File: pkg/policy/policy.go [hunk 1]
```diff
@@ -63,6 +66,26 @@ func (p Policy) TotalQueries() int {
 	return count + len(p.Checks)
 }
 
+func (p Policy) Filter(path string) Policy {
```
Comment (roneli) [inline pkg/policy/policy.go:69]: I feel like this can also be used in the executor to find a policy / check, wdyt?

**Review by roneli — APPROVED** [review]
LGTM, had one suggestion/nit, since I think the selector parsing is similar to in the executor I think we can use this function there (at least for subpolicies and not checks). Make tests pass and answer my comment and we can merge.

File: pkg/policy/policy.go (hunk 1 above)
Comment (disq) [reply pkg/policy/policy.go:69]: I too think all filtering should be unified so that if (erm, when) it's buggy, it's buggy the same way 😂

Comment (bbernays) [reply pkg/policy/policy.go:69]: Added support for filtering checks

**Pull request merged**